source/HitboxAnimationManager.hx

```diff
@@ -0,0 +1,125 @@
+package;
```

Comment:
Fun thing I learned about Github with that:  You can paste a screenshot straight from the clipboard into a comment like this, and it'll upload and insert it automatically.  QoL right there.

Also, I'm not sure why I didn't note this before, but the sword image on the player should probably go `visible = false` during the swing animation.